### ISSUE TITLE
Enhance the filesystem

### ISSUE THREAD

**Issue by RuiL1904** [coderdojobraga/gongo#24]
I think the filesystem it's not the best right now. For example, `constants.py` and `belts.json` are inside the `cogs` folder (which is designed to handle the bot cogs) and the files generated by the database are not inside a proper folder (called `data` for instance). I think we could follow a conventional filesystem, but I'm not that deep into this topic.
What's your opinion about this, @danielsp45 and @nelsonmestevao?

**Comment by danielsp45** [comment]
Yes I agree with you, but I also don't know the best way to organize them.
